lib/kuzzle/kuzzle.js

```diff
@@ -173,8 +173,12 @@ class Kuzzle extends KuzzleEventEmitter {
 
       await this.validation.curateSpecification();
 
+      await this.internalIndex.updateMapping('users', options.import.userMappings);
+
       await this.ask('core:storage:public:mappings:import', options.mappings);
 
+      await this.ask('core:storage:public:mappings:import', options.import.mappings);
```

Comment:
We should ensure that users are not trying to use both `import.*` and the deprecated way with `mappings`